index.html

```diff
@@ -22,33 +22,49 @@
       width: 80%;
       max-width: 320px;
     }
-
-    .app__init-logo {
+    .app__init-left-face-logo {
+      position: absolute;
       margin: 0 auto;
       width: 100%;
+      animation: init-left-face-logo .6s ease-out infinite;
     }
-
-    .app__init-load-spinner {
-      width: 60px;
-      height: 60px;
+    .app__init-top-face-logo {
+      position: absolute;
       margin: 0 auto;
+      width: 100%;
+      animation: init-top-face-logo .6s ease-out infinite;
     }
-
-    .app__init-load-spinner:after {
-      content: "";
-      display: block;
-      width: 40px;
-      height: 40px;
-      border-radius: 50%;
-      border: 8px solid;
-      border-color: #3a4180 transparent;
-      animation: init-load-spinner 1.2s linear infinite;
+    .app__init-right-face-logo {
```

Comment:
left right and top face are mostly common, extract common code into like
```css
.app__init-right-face-logo,
.app__init-left-face-logo,
.app__init-top-face-logo {
      position: absolute;
      margin: 0 auto;
      width: 100%;
}
.app__init-left-face-logo {
  animation: /* ... */
}
```

Reply:
ok